src/datagrid/DataGridFilter.tsx

```diff
@@ -237,9 +249,10 @@ export class DataGridFilter extends React.PureComponent<DataGridFilterProps, Dat
                             name={`name-${columnName}`}
                             placeholder={placeholder}
                             defaultValue={filterValue}
-                            onChange={evt => {
-                                this.updateFilter(evt.target.value);
-                            }}
+                            disabled={disabled || false}
+                            onChange={(evt: React.ChangeEvent<HTMLInputElement>) =>
```

Comment:
can we add a prop to enable/disable `debounce `effect ?

Reply:
Added now. I dont expect to users to use it.  Default 500ms is good for standard usecases. But good to have to have option if we want to disable it.